api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -102,12 +102,28 @@
  * <em>Implicit</em> converters are only created when no other converter was found; therefore, they do not have a
  * priority.
  *
+ * <h3 id="empty">Empty values</h3>
+ *
+ * For all converters, the empty string {@code ""} <em>must</em> be considered an empty value. Some converters <em>may</em>
```

Comment:
I have not fully reviewed whether this changes any logic. But please be aware, that changing this logic will also have an effect on default values within mp-config but also business apps. It might be a hidden API change.

Reply:
This does change logic.  But it's not a hidden change; on the contrary it's spelled out clearly for the first time.